src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/template.json

```
@@ -7,7 +7,7 @@
     "WebAssembly"
   ],
   "name": "Blazor WebAssembly App",
-  "defaultName": "WebApplication",
+  "defaultName": "BlazorApplication",
```

```suggestion
  "defaultName": "BlazorApp",
```

For consistency with https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json#L448